pkg/common/config/config.go

```diff
@@ -293,6 +301,17 @@ func validateConfig(ctx context.Context, cfg *Config) error {
 		}
 	}
 
+	if cfg.FeatureStatesConfig.Name == "" && cfg.FeatureStatesConfig.Namespace == "" {
+		cfg.FeatureStatesConfig.Name = DefaultFSSConfigMapName
+		if GetClusterFlavor(ctx) == cnstypes.CnsClusterFlavorVanilla {
```

Comment:
save retrieved cluster flavor in the local variable, instead of calling multiple times in this function.

Reply:
Done